example/run.sh

```diff
@@ -109,10 +109,8 @@ echo ""
 python $root_dir/main.py --train \
     --train_files $out_dir/train_data.h5 \
     --val_files $out_dir/val_data.h5 \
-    --model resnet --nblocks 5 --nfilt 15 --width 50 \
```

Comment:
for the run.sh file, let's explicitly specify the `model_config` option to make it clearer for users

Reply:
done.